### PR TITLE
Remove v1 references

### PR DESCRIPTION
**Pull request by diondrapeck**
# Description

To focus our documentation offerings, this PR removes all v1 content from this repository. v1 content is only recommended for existing v1 users, and all new users should use v2. For existing v1 users, the v1 samples repository is where they can find remaining v1 samples: https://github.com/Azure/MachineLearningNotebooks. Note that the v1 repository is no longer updated or monitored because v1 is not being updated with new features.

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: v1/scripts/validation/check_notebook_output.py
Comment (kdestin) [inline v1/scripts/validation/check_notebook_output.py:1]: Some of the v2 automl notebooks might use this to check output? @jeff-shepherd  is this safe to remove?

File: cli/jobs/nebulaml/cifar10_deepspeed/README.md [hunk 1]
```diff
@@ -18,10 +18,6 @@ is using Deepspeed, you will need to have
 - ACPT image in the environment. See [Azure Container for PyTorch - Azure Machine Learning | Microsoft Learn](https://learn.microsoft.com/en-us/azure/machine-learning/resource-azure-container-for-pytorch) for more details about ACPT image.
 - DeepSpeed version is no late than 0.7.3, which all the services of Nebula have already be integrated.
 
-## Original Example Repo
-
-In this tutorial, we use an example code of AzureML with DeepSpeed that trains a simple CNN model on CIFAR. You can find the original repo [over here](../../../../v1/python-sdk/workflows/train/deepspeed/cifar).
-
```
Comment (kdestin) [inline cli/jobs/nebulaml/cifar10_deepspeed/README.md:24]: Does this sample depend on the this file?

Comment (diondrapeck) [reply cli/jobs/nebulaml/cifar10_deepspeed/README.md:24]: None of the files reference v1 except for that deleted segment in the README

File: v1/scripts/validation/check_notebook_output.py
Comment (diondrapeck) [reply v1/scripts/validation/check_notebook_output.py:1]: Yeah, there's also a `check_v2_experiment_result.py` file inside this v1/scripts/validation folder. Waiting for Jeff's opinion to see if we need to move these outside the v1 folder.

**Review by jeff-shepherd — APPROVED** [review]

File: v1/scripts/validation/check_notebook_output.py
Comment (jeff-shepherd) [reply v1/scripts/validation/check_notebook_output.py:1]: @kdestin @diondrapeck I already moved the check output and other scripts needed for V2 to azureml-examples\.github\test\scripts.

So, it is safe to remove v1/scripts/validation folder.

**Review by kdestin — APPROVED** [review]

**Pull request merged**